src/hotspot/cpu/riscv/gc/z/zBarrierSetAssembler_riscv.cpp

```diff
@@ -442,9 +442,9 @@ void ZBarrierSetAssembler::generate_c1_load_barrier_runtime_stub(StubAssembler*
   __ epilogue();
 }
 
-#undef __
 #endif // COMPILER1
 
+#undef __
```

Comment:
I think we should remove the "#undef __" at line #354 of this file at the same time.

Reply:
Good catch!

Updated.
Thanks.